Dockerfile

```diff
@@ -14,9 +15,11 @@ WORKDIR /home/$OSCAP_USERNAME
 
 COPY . $OSCAP_DIR/
 
+# clean the build dir in case the user is also building SSG locally
+RUN rm -rf $OSCAP_DIR/build/*
+
 WORKDIR /home/$OSCAP_USERNAME/$OSCAP_DIR/build
 
 RUN cmake ..
 
-ENTRYPOINT ["/usr/bin/make"]
-CMD ["all"]
+CMD /usr/bin/make -j $BUILD_JOBS
```

Comment:
@redhatrises yeah, unfortunately, this is necessary if you want to use env vars in `CMD`.